andela/travel_tool_front
# Analytics tests coverage increment

#### What does this PR do?
Increases test coverage on the analytics and fix bugs on analytics

#### Description of Task to be completed?
- Write tests for untested features 
- Match displayed data with the corresponding titles
- Match the current location with the displayed data

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
```fx-dashboard-tests```
- Install dependencies:
```yarn install```
`yarn test`
- All the tests should be passing
- Start the application:
```yarn start```

- Login to the application
- Navigate to the dashboard page on the sidebar
- You should be able to get results as shown in the screenshot below.

#### Any background context you want to provide?
1. To use the Dashboard - Create your standalone component and plug it into the dashboard component.
2. You should have the user role of a Travel Administrator or Super Administrator to be able to see the `manage` page. To do that:
  - Download `Postico` and connect to your travela database with relevant details: Db name, password, port, user
  - Click `userRoles` table and set the `roleId` to that of a Travel Administrator(29187) or Super Administrator(10948) 

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
<img width="721" alt="screen shot 2018-11-05 at 17 58 07" src="https://user-images.githubusercontent.com/22371596/48005444-828ab200-e124-11e8-88c0-90cebad3d511.png">
